cvmfs/swissknife_sync.h

```diff
@@ -28,7 +28,10 @@ struct SyncParameters {
     avg_file_chunk_size(8*1024*1024),
     max_file_chunk_size(16*1024*1024),
     manual_revision(0),
-    max_concurrent_write_jobs(0) {}
+    max_concurrent_write_jobs(0),
+    is_balanced(false),
+    max_weight(100000),
```

Comment:
There are other cases like this one in that constructor. Should I change them, or maybe this is another pull request's stuff?